thrift/linkattacher/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+# Copyright (C) 2016 Istituto Italiano di Tecnologia iCub Facility & RBCS Department
+# Authors: Yeshasvi Tirupachuri
+# CopyPolicy: Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
+
+cmake_minimum_required(VERSION 2.8.11)
```

Comment:
You can safely use `3.5`

Reply:
done. Also, updated it for the plugin CMakeLists.txt